crypto/evp_extra/p_pqdsa_asn1.c

```diff
@@ -129,6 +132,30 @@ static int pqdsa_pub_encode(CBB *out, const EVP_PKEY *pkey) {
   return 1;
 }
 
+static int pqdsa_priv_encode_seed(CBB *out, const EVP_PKEY *pkey) {
+  PQDSA_KEY *key = pkey->pkey.pqdsa_key;
+  const PQDSA *pqdsa = key->pqdsa;
+  if (key->seed == NULL) {
+    OPENSSL_PUT_ERROR(EVP, EVP_R_ENCODE_ERROR);
+    return 0;
+  }
+  // See https://datatracker.ietf.org/doc/draft-ietf-lamps-dilithium-certificates/ section 6.
+  CBB pkcs8, algorithm, oid, seed;
+  if (!CBB_add_asn1(out, &pkcs8, CBS_ASN1_SEQUENCE) ||
+      !CBB_add_asn1_uint64(&pkcs8, 0 /* version */) ||
```

Comment:
```suggestion
      !CBB_add_asn1_uint64(&pkcs8, PKCS8_VERSION_ONE) ||
```

Reply:
Also out of general curiosity I know the draft standard uses PKCS#8 v2 encoding so that it can include the public key. Is there a reason we aren't supporting serialization of v2 as well? (I can see why we would want to support the v1 encoding as well). I guess why not both?

Reply:
No reason other than PR scope and the original request --  I wanted to add try to keep this PR to a single feature.

Reply:
Support full v2 encoding added in [9e86f22](https://github.com/aws/aws-lc/pull/2194/commits/9e86f22d1411ac6f5016f12cfb479616ee4e094a)